ballista/rust/executor/src/executor_server.rs

```diff
@@ -164,8 +152,8 @@ impl ExecutorServer {
         // TODO Error handling
         self.scheduler
             .clone()
-            .send_heart_beat(SendHeartBeatParams {
-                metadata: Some(self.executor_meta.clone()),
+            .receive_heart_beat(HeartBeatParams {
+                executor_id: self.executor.metadata.id.clone(),
                 state: Some(self.get_executor_state().await.into()),
             })
             .await
```

Comment:
maybe it is good to rename the method name to heart_beat_from_executor().

Reply:
Agree. It's more clear than before.